website/views/options/social/group.jade

```diff
@@ -17,6 +17,8 @@ a.pull-right.gem-wallet(ng-if='group.type!="party"', popover-trigger='mouseenter
         .panel-heading(bindonce='group')
           h3.panel-title
             | {{group.name}}
+            span(ng-if='group')
```

Comment:
`ng-if` is necessary here so that the isMemberOfGroup doesn't get called until after the group has actually loaded.